backend/tournesol/views/polls.py

```diff
@@ -201,3 +202,16 @@ def get_queryset(self):
         queryset = self.annotate_with_total_score(queryset, self.request, poll)
         queryset = self.filter_unsafe(queryset, filters)
         return queryset.order_by("-total_score", "-pk")
+
+
+class PollsCriteraScoreDistributionView(PollScopedViewMixin, RetrieveAPIView):
```

Comment:
I think a docstring here would help for the Swagger UI


![Screenshot 2022-04-26 at 09 53 36](https://user-images.githubusercontent.com/8818081/165250546-0c8d4b75-53a5-4532-85f4-cc3d71d5c599.png)

Reply:
It's added